ee/conftest.py

```diff
@@ -32,9 +71,7 @@ def django_db_setup(django_db_setup, django_db_keepdb):
     if not django_db_keepdb or not database.db_exists:
         database.create_database()
 
-    database.migrate("ee.clickhouse.migrations")
-    # Make DELETE / UPDATE synchronous to avoid flaky tests
-    sync_execute("SET mutations_sync = 1")
```

Comment:
Not needed anymore. the delete_person test was solved by setting the configuration for clickhouse overall [here](https://github.com/PostHog/posthog/pull/5732)